### ISSUE TITLE
Can you share a simplest test scene? even a box

### ISSUE THREAD

**Issue by aaronmack** [pablode/gatling#8]
I got the following error while running

`Exception generating MDL code: Could not find a nodedef for shadernode 'SR_Invalid'`

My guess is that my scene is not built correctly.

Any help would be greatly appreciated.

Aaron

**Comment by pablode** [comment]
Hi Aaron!

There's a [cornell.usdc](https://github.com/pablode/gatling/blob/master/cornell.usdc) file in the root folder.

Three more assets can be found in my [usd-assets](https://github.com/pablode/usd-assets) repository.

You can try to render them with `usdview` or the gatling executable.

Hope this helps!

**Comment by aaronmack** [comment]
Hi Pablo!
Thanks for your help.

I always get errors like below when running and get `core dumped`.
```
> usdview usd-assets/cornell-box/mdl/cornell-box.usda
Exception generating MDL code: Could not find a nodedef for shadernode 'SR_Invalid'
Building geom cache for camera /_UsdImaging_HdGatlingRendererPlugin_0x7f4eb81f6200/camera
#Vertices: 96
#Faces: 32
Segmentation fault (core dumped)
```
OS: Ubuntu 20.04
CMake: 
-DUSD_ROOT=/home/test/USD-build -DMDL_ROOT=/home/test/mdl-sdk-349500.8766a -DCMAKE_INSTALL_PREFIX=/home/test/USD-build/plugin/usd -DCMAKE_BUILD_TYPE=Release
USD: dev

**Comment by aaronmack** [comment]
Or get

`[fatal] (shadergen) MaterialX MDL file path not found, translation not possible`

When I cleared the entire USD-build directory and recompiled everything.

**Comment by pablode** [comment]
Thanks for the info, I think I know what's going on.

It's related to a recent change, which I tested without clearing the install files. You should be able to fix it by copying some folders manually:
- copy `extern/MaterialX/libraries` to `<USD_INSTALL>/plugin/usd/hdGatling/resources/mtlxlib`
- copy `extern/MaterialX/source/MaterialXGenMdl/mdl/materialx` to `<USD_INSTALL>/plugin/usd/hdGatling/resources/mtlxmdl`

I'll implement a proper fix later and make sure it works. Thanks for bringing this up!

**Comment by pablode** [comment]
Fixed in 4b074df.

**Comment by aaronmack** [comment]
Works correctly, thanks
